README.md

```diff
@@ -33,6 +33,7 @@ Defaults to `password`.
   This is a dict of the form of `KEY: VALUE`.
 * `size`: (optional) size to make the image filesystem.
 * `properties`: (optional) dict of properties to set on the glance image.
+* `type`: (optional) image type. Default in DIB is qcow2.
```

Comment:
Nit - it might help to describe the dictionary of valid options here, or where it could be found.